darwin/amplitude_flutter.podspec

```diff
@@ -32,5 +32,5 @@ The official Amplitude Flutter SDK for tracking analytics events in your Flutter
     'DEBUG_INFORMATION_FORMAT' => 'dwarf-with-dsym'
   }
   s.swift_version = '5.9'
-  s.dependency 'AmplitudeSwift', '~> 1.14'
+  s.dependency 'AmplitudeSwift', '~> 1.15'
```

Comment:
Note: AmplitudeSwift doesn't actually have v1.15 yet, so cannot merge this PR until that is released